tests/core/daemon/test_daemon.py

```diff
@@ -777,3 +777,49 @@ async def test_key_label_methods(
     keychain.add_private_key(test_key_data.mnemonic_str(), "key_0")
     await ws.send_str(create_payload(method, parameter, "test", "daemon"))
     assert_response(await ws.receive(), response_data_dict)
+
+
+@pytest.mark.asyncio
+async def test_bad_json(daemon_connection_and_temp_keychain: Tuple[aiohttp.ClientWebSocketResponse, Keychain]) -> None:
+    ws, _ = daemon_connection_and_temp_keychain
+
+    await ws.send_str("{doo: '12'}")  # send some bad json
+    response = await ws.receive()
+
+    # check for error response
+    assert response.type == aiohttp.WSMsgType.TEXT
+    message = json.loads(response.data.strip())
+    assert message["data"]["success"] is False
+    assert message["data"]["error"].startswith("Expecting property name")
+
+    # properly register a service
+    service_name = "test_service"
+    data = {"service": service_name}
+    payload = create_payload("register_service", data, service_name, "daemon")
+    await ws.send_str(payload)
+    await ws.receive()
```

Comment:
Worth making sure the response indicates success so as to catch any issues here early and avoid confusion by an error further down?

Reply:
IMO no